docs/howto/custom-template-tags.txt

```diff
@@ -199,11 +199,10 @@ passed around inside the template code:
   They're commonly used for output that contains raw HTML that is intended
   to be interpreted as-is on the client side.
 
-  Internally, these strings are of type ``SafeBytes`` or ``SafeText``.
-  They share a common base class of ``SafeData``, so you can test
-  for them using code like::
+  Internally, these strings are of type ``SafeText``. You can test for them
```

Comment:
Could you make this a link and add an import for `SafeText` in the code example?